src/python/deepgnn/graph_engine/snark/converter/json_converter.py

```diff
@@ -455,27 +455,73 @@ def __add_sparse(node, feature, tp, container):
             + bytes(coordinates.data)
             + values_buf
         )
+
+        if int(k) in container:
+            newId = __get_unassigned_index(node)
+            get_logger().warning(
+                f"Duplicate feature ids found for a node, original: {k}, replaced with: {newId}"
+            )
+            k = newId
+
```

Comment:
Why do we not require users to submit the correct/new ids by themselves?

Reply:
Sometimes it is easy to make mistake when assigning the feature index, and the converter process will be pending there. This issue is discussed with LinkedIn team in order to make the converter process easier for them.